packages/api/src/Base.ts

```diff
@@ -331,7 +331,7 @@ export default abstract class ApiBase<CodecResult, SubscriptionResult> implement
     this._eventemitter.emit(type, ...args);
   }
 
-  private init (): void {
+  private init (genesisSpecToMetaDataMap: {[key: string]: string}): void {
```

Comment:
no need to change signature of init(), you can use this.options to get visit the value

Reply:
same for loadMeta()